apps/pie-storybook/decorators/component-status.ts

```diff
@@ -0,0 +1,30 @@
+import { TemplateResult, html } from 'lit';
+import '@justeattakeaway/pie-tag';
+import { StorybookContext } from '../interfaces/storybook-context';
+import componentStatusesJson from '../component-statuses.json';
+import { ComponentStatuses } from '../interfaces/component-statuses';
+import { tagVariantToStatusMap } from '../data/tag-variants-to-statuses-map';
+
+const componentStatuses: ComponentStatuses = componentStatusesJson;
+
+export const ComponentStatus = (story: () => TemplateResult, storybookContext: StorybookContext) => {
+    const componentStatus: string = componentStatuses[storybookContext.component];
+    const tagVariant = tagVariantToStatusMap[componentStatus];
+    const position = storybookContext.parameters.componentStatusPosition || 'bottom';
+
+    if (storybookContext.viewMode === 'docs' || !componentStatus || !tagVariant) {
```

Comment:
I think a comment explaining `storybookContext.viewMode === 'docs'` might be helpful

Reply:
done!